badges/badges1.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="badges1.css">
+    <link rel="stylesheet" href="/Users/raranjan/Documents/component library/component-librarys/clhome.css">
```

Comment:
Have a look at the path of the second css file. You can move the file to the project's folder, like you did the first css file.